tools/common.sh

```diff
@@ -39,6 +39,24 @@ if [ -z "$APPENGINE_DIR" ]; then
     exit 1
 fi
 
+for dir in \
```

Comment:
@gimite I implemented this before we agree on the decision to delete tools/gae.  It might be usable in the future but not used now. Please let me know if you prefer to delete it.

Reply:
I prefer to delete this one if we are currently not using it.

But how about $APPENGINE_DIR? I guess it's still necessary to run some scripts, right? Do you expect developers to manually set $APPENGINE_DIR? Or should we update their defaults to point to the default location of AppEngine SDK in Cloud SDK?